docs/setup/new_environment.md

```diff
@@ -1,162 +1,223 @@
+
+    Uncomment and set the value of `ufw_private_interface` to the network interface of your machine that we found in the previous step.
+
 
 ### Add a public key and user to commcare-cloud
 
 Even though we will be running all commands locally, we still need to add the user's public key.
 
 1. Generate a public key (optional):
 
- If your user already has an ssh key pair, ignore this step.
+    If your user already has an ssh key pair, ignore this step.
 
- ``` bash
- $ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
- ```
+    ``` bash
+    $ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+    ```
 
 1. Add your public key to commcare-cloud by copying it to `~/environments/_authorized_keys`
 
- For example: 
- ``` bash
- $ cp ~/.ssh/id_rsa.pub ~/environments/_authorized_keys/$(whoami).pub
- ```
+    For example:
+    ``` bash
+    $ cp ~/.ssh/id_rsa.pub ~/environments/_authorized_keys/$(whoami).pub
+    ```
 
 1. Add your system username to the `present` section of `~/environments/_users/dimagi.yml`. This username should correspond to the name you've used for the public key in the previous step.
 
 
-## Step 3
+## Step 3: Install commcare-cloud
 
-Install commcare-cloud onto the monolith as described in ./installation.md beginning with step 2.
+Install commcare-cloud onto the monolith as described in [Installing CommCareHQ](installation.md#step-2) beginning with **Step 2**.
 
-## Step 4
+## Step 4: Set-up commcare-cloud
 
-```bash
-COMMCARE_CLOUD_ENVIRONMENTS=/home/{username}/environments manage-commcare-cloud configure
+1. Configure `commcare-cloud`
 
-#Let's get you set up to run commcare-cloud.
+    ``` bash
+    $ COMMCARE_CLOUD_ENVIRONMENTS=/home/$(whoami)/environments manage-commcare-cloud configure
 
-Do you work or contract for Dimagi? [y/N]n
+    ```
+    You will see a few prompts that will guide you through the installation. Answer the questions as follows for a standard installation:
+    ```
+    Do you work or contract for Dimagi? [y/N]n
+    ```
 
-To use commcare-cloud, you have to have an environments directory. This is where you will store information about your cluster setup, such as the IP addresses of the hosts in your cluster, how different services are distributed across the machines, and all settings specific to your CommCare instance.
-I see you have COMMCARE_CLOUD_ENVIRONMENTS set to /home/{username}/environments in your environment
-Would you like to use environments at that location? [y/N]y
+    ```
+    I see you have COMMCARE_CLOUD_ENVIRONMENTS set to /home/{username}/environments in your environment
+    Would you like to use environments at that location? [y/N]y
+    ```
 
-echo "source ~/.commcare-cloud/load_config.sh" >> ~/.profile
-```
+1. Add the commcare-cloud config to your bash profile to set the correct paths:
+
+    ``` bash
+    $ echo "source ~/.commcare-cloud/load_config.sh" >> ~/.profile
+    ```
+
+1. Log out of your machine and log in again so these changes take effect.
```

Comment:
I thought that these change happened automatically after running configure, but probably just making that up.

I think you can also run either `source ~/.commcare-cloud/load_config.sh` or `source init-ansible` (not sure on the exact script for that) to have the same effect if you want to avoid log out.

Reply:
I don't know why, but if you don't log out and log in again, `commcare-cloud` doesn't take into account the environment you set up in the previous step, so running `commcare-cloud monolith` commands fails.
I haven't had time to debug this, but maybe @dannyroberts has some ideas. 


``` bash
(ansible) farid@monolith:~/commcare-cloud$ echo "source ~/.commcare-cloud/load_config.sh" >> ~/.profile
(ansible) farid@monolith:~/commcare-cloud$ commcare-cloud monolith update-local-known-hosts
usage: commcare-cloud [-h] [--control]
                      
                      {64-test,development,echis,enikshay-reference,icds,icds-cas,icds-new,india,pna,production,staging,swiss}
                      {bootstrap-users,ansible-playbook,django-manage,aps,aws-sign-in,tmux,ap,validate-environment-settings,openvpn-activate-user,deploy-stack,service,update-supervisor-confs,update-users,ping,migrate_couchdb,lookup,run-module,update-config,copy-files,mosh,list-postgresql-dbs,after-reboot,ssh,downtime,fab,update-local-known-hosts,send-datadog-event,aws-list,aws-fill-inventory,migrate-couchdb,terraform,openvpn-claim-user,celery-resource-report,run-shell-command,terraform-migrate-state}
                      ...
commcare-cloud: error: argument env_name: invalid choice: 'monolith' (choose from '64-test', 'development', 'echis', 'enikshay-reference', 'icds', 'icds-cas', 'icds-new', 'india', 'pna', 'production', 'staging', 'swiss')
```